### PR TITLE
Add Logo page

### PR DESCRIPTION
**Pull request by tringuyenanduin**
This is 99% done with 1 single problem that i can't solve. Maybe @dvkndn can help me look in to it. 

The URL is correctly added but no image shows up. Not sure why

screenshot 2018-09-04 17 57 18

### CODE REVIEW

File: core/src/main/scala/anduin/guide/page/PageLogo.scala [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import anduin.component.icon.Icon
+import anduin.guide.{Pages, Router}
+import anduin.mcro.Source
+import anduin.style.Style
+
+object PageLogo {
+
+  def renderLink(fileNameAndExtension: String): String = {
+    s"https://s3.amazonaws.com/anduin-static-assets/gondor/logos/$fileNameAndExtension"
+  }
```
Comment (dvkndn) [inline core/src/main/scala/anduin/guide/page/PageLogo.scala]: static values are better to be `val` or `lazy val`. `def` will evaluate it each time

Comment (dvkndn) [reply core/src/main/scala/anduin/guide/page/PageLogo.scala]: Also this should be `private`

File: core/src/main/scala/anduin/guide/page/PageLogo.scala [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+  def renderLogo(fileName: String, downloadName: String): VdomElement = {
+    <.div(
+      ^.width := "50%",
```
Comment (dvkndn) [inline core/src/main/scala/anduin/guide/page/PageLogo.scala]: This is width.pc50

File: core/src/main/scala/anduin/guide/page/PageLogo.scala [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      ^.height := "160px",
+      ^.paddingBottom := "40px",
+      Style.border.all.borderWidth.px1.borderColor.gray3.position.relative.textAlign.center,
```
Comment (dvkndn) [inline core/src/main/scala/anduin/guide/page/PageLogo.scala]: You should break line..

File: core/src/main/scala/anduin/guide/page/PageLogo.scala [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+      <.img(
+        Style.flexbox.none,
+        ^.src := renderLink(s"$fileName.svg"),
+        TagMod(^.width := "278", ^.height := "278")
+      ),
+      <.a(
+        ^.bottom := "0",
+        ^.left := "0",
+        ^.right := "0",
```
Comment (dvkndn) [inline core/src/main/scala/anduin/guide/page/PageLogo.scala]: These are available under Style.coordinate

File: core/src/main/scala/anduin/guide/page/PageLogo.scala [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+      ),
+
+      // Single logo row
+      Markdown(
+        """
+          |# Default
+        """.stripMargin
+      )(),
+      <.div(
+        ^.cls := "row",
```
Comment (dvkndn) [inline core/src/main/scala/anduin/guide/page/PageLogo.scala]: How the hell could this "row" work..

**Comment by tringuyenanduin** [comment]
Good to know all of your comment. Was coding straight from my Sublime, so no support for style check and surprisingly it can still build successfully. 

Thank you anyway @dvkndn

**Pull request merged**